src/EnvironmentVariables.jsx

```diff
@@ -1 +1,3 @@
 export const BASE_URL = "http://localhost:8000";
+export const FETCH_CITY_URL =
```

Comment:
1. File extension should be `.js` since it doesn't contain any UI.
2. Environment variables is smth different. They should be placed in `.env.development` or `.env.production` files and, depending on the build environment, corresponding file is being picked up. Each file contains different set of variables. What you have here is just constants.
You can read about env variables here - https://create-react-app.dev/docs/adding-custom-environment-variables/